config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presubmits-main.yaml

```diff
@@ -55,21 +55,23 @@ presubmits:
           - ./scripts/ci-e2e.sh
         env:
           - name: GINKGO_FOCUS
-            value: "Workload cluster creation"
+            value: \[REQUIRED\]
           - name: GINKGO_SKIP
-            value: "Creating a GPU-enabled cluster|.*Windows.*dockershim.*|.*AKS.*|Creating a cluster that uses the external cloud provider"
+            value: ""
         # docker-in-docker needs privileged mode
         securityContext:
           privileged: true
     annotations:
       testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-azure
       testgrid-tab-name: capz-pr-e2e-main
       testgrid-alert-email: k8s-infra-staging-cluster-api-azure@kubernetes.io
-  - name: pull-cluster-api-provider-azure-e2e-full
+  - name: pull-cluster-api-provider-azure-e2e-optional
```

Comment:
should we also change the test grid tag https://github.com/kubernetes/test-infra/blob/f40470e829b3d819e4c7a63864d4957d557b49f9/config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presubmits-main.yaml#L100

Reply:
Can we rely upon testgrid to "garbage collect" the old tab name if we change it? Would make sense to rename it, yeah.

Reply:
yeah, +1 to renaming the tab, missed this

testgrid should handle this for us, we'll just loose job history (which we would anyways by renaming it) AFAIK

Reply:
done